charts/sonarqube-dce/templates/prometheus-podmonitor.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if and .Values.ApplicationNodes.prometheusExporter.enabled .Values.ApplicationNodes.prometheusExporter.podMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ template "sonarqube.name" . }}
+  namespace: {{ .Values.ApplicationNodes.prometheusExporter.podMonitor.namespace | quote }}
+  labels:
+    app: {{ template "sonarqube.name" . }}
+spec:
+   {{- if  .Values.ApplicationNodes.prometheusExporter.podMonitor.jobLabel }}
+  jobLabel: {{ .Values.ApplicationNodes.prometheusExporter.podMonitor.jobLabel | quote }}
+  {{- end }}
+  namespaceSelector:
+    matchNames:
+    - {{ .Release.Namespace }}
+  selector:
+    matchLabels:
+      app: {{ template "sonarqube.name" . }}
+  podMetricsEndpoints:
+  - port: monitoring-ce
+    path: /
```

Comment:
Are we sure this '/' path will always be true ? 

Is there an SQ parameter that could modify that ? probably not that kind of stuff is handled by the reverse proxy i believe ?

Reply:
I do not think there is a property that modifies it. But, while you were talking about this, I also realize that `monitoring-ce` and `monitoring-web` do not really expose prometheus metrics. Shall we keep just the real prometheus endpoint in the resource (i.e., `/api/monitoring/metrics`) as we specify in the [docs](https://docs.sonarqube.org/latest/instance-administration/monitoring/)?

Reply:
Yes there is something in the picture i still do not understand.

We have an init container that inject the prometheus exporter, that is no longer needed i believe, as it is now fully included in SQ no ? 

That's probably how it was done before, so yup i would agree we remove every mention to those things.

Reply:
It's indeed a tricky topic. As discussed, we both need to convert the JMX metrics and support the native prometheus metrics. For now, I was just decoupling the prometheus exporter and the prometheus PodMonitor. In my opinion, they are independent. Furthermore, I made sure that our podmonitor only supports the native prometheus endpoint. We need to discuss how to support such Prometheus exporter in the future...